articles/text-inputs.mdx

```diff
@@ -8,127 +8,247 @@ published: true
 image: >-
   https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
 description: >-
-  Details for setting up and using different types of plain text inputs for
-  editing your data.
+  Learn how to configure a Text input to store plain text content in your data
+  files or front matter.
+article_category:
+  - Reference
+article_topic:
+  - Inputs
 tags: []
 related_articles: []
 related_links: []
 ---
-These inputs are for editing plain text in your data.
+A text input lets your team members add plain text to your data files or front matter.
+
+There are four types of Text input:
+
+* Text
+* Textarea
+* Email
+* Disabled
+
+For each input, you can configure a character counter (Textarea only) and determine how CloudCannon handles null values. You can also use the [general configuration options](/documentation/articles/configure-your-inputs/#general-configuration-options) available for all inputs.
```

Comment:
"For each input, you can configure a character counter (Textarea only)" reads a bit clunkily

```suggestion
For Textarea inputs, you can configure a character counter. For each input, you can configure how CloudCannon handles null values. You can also use the [general configuration options](/documentation/articles/configure-your-inputs/#general-configuration-options) available for all inputs.
```